README.md

```diff
@@ -199,6 +206,12 @@ Example
     python analyze.py --train_file_path=~/tacotron/training/train.txt --save_to=~tacotron/visuals --cmu_dict_path=~/cmudict-0.7b
 ```
 
+In case of docker usage you've to export your DISPLAY variable on container start to successfully create the image files. Ensure x server connections are enabled on your host system.
```

Comment:
Perhaps slightly rewording to:
> If using Docker, you need to export your DISPLAY variable when starting the container to successfully create the image files. Also ensure that X server connections are enabled on your host system.

Reply:
I will change the text to your proposal.